invenio/modules/accounts/upgrades/accounts_2015_06_26_userext.py

```diff
@@ -0,0 +1,54 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of Invenio.
+# Copyright (C) 2015 CERN.
+#
+# Invenio is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 2 of the
+# License, or (at your option) any later version.
+#
+# Invenio is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Invenio; if not, write to the Free Software Foundation, Inc.,
+# 59 Temple Place, Suite 330, Boston, MA 02111-1307, USA.
+
+"""Change id column from VARBINARY to String."""
```

Comment:
A general comment for the future: when a user sees an upgrade recipe message saying:

- Change id column from VARBINARY to String.

that it may not be very informative.  It's better to be more specific and at least repeat the table name.  Here are some examples of the messages we used in the past in these situations:

- New xtrJOB.last_recid column
- Larger values allowed for seqSTORE.seq_value